src/common/docsEnvironmentController.ts

```diff
@@ -27,16 +36,29 @@ export class DocsEnvironmentController implements EnvironmentController, vscode.
         return this.environment;
     }
 
+    public get docsRepoType(): DocsRepoType {
+        return this._docsRepoType;
+    }

```

Comment:
no corresponding setter because ts require setter and getter having same accessibility